ini.h

```diff
@@ -127,6 +127,12 @@ int ini_parse_string(const char* string, ini_handler handler, void* user);
 #define INI_STOP_ON_FIRST_ERROR 0
 #endif
 
+/* Detect section headers (default is to NOT detect entering new sections,
+   like ConfigParser) */
```

Comment:
Good call, thanks. Can you please change the comment to be a bit more specific. How about this?

```c
/* Nonzero to call the handler at the start of each new section (with
   name and value NULL). Default is to only call the handler on
   each name=value pair. */
```

Reply:
Done, thanks for the wording suggestion!